commonui/src/test/java/com/hyperwallet/android/ui/common/util/TransferMethodUtilsTest.java

```diff
@@ -151,29 +152,31 @@ public void getTransferMethodDetail_returnsPayPalDetails() {
                 "sunshine.carreiro@hyperwallet.com").build();
 
         String actual = getTransferMethodDetail(mContext, transferMethod, PAYPAL_ACCOUNT);
-        assertThat(actual, is("sunshine.carreiro@hyperwallet.com"));
+        MatcherAssert.assertThat(actual, Matchers.is("sunshine.carreiro@hyperwallet.com"));
     }
 
     @Test
     public void getTransferMethodDetail_returnsCardDetails() {
         HyperwalletTransferMethod transferMethod = new HyperwalletBankCard.Builder().cardNumber(
                 "************0006").build();
 
-        when(mContext.getString(eq(R.string.transfer_method_list_item_description), eq("0006"))).thenReturn(
+        when(mContext.getString(ArgumentMatchers.eq(R.string.transfer_method_list_item_description),
+                eq("0006"))).thenReturn(
                 "Ending on 0006");
         String actual = getTransferMethodDetail(mContext, transferMethod, BANK_CARD);
-        assertThat(actual, is("Ending on 0006"));
+        MatcherAssert.assertThat(actual, Matchers.is("Ending on 0006"));
```

Comment:
please import the whole static class for `MatcherAssert` and `Matchers` please

Reply:
removed